src/Controller/Activity/ActivityController.php

```diff
@@ -174,35 +219,39 @@ public function registerAction(Request $request, Activity $activity, MailService
     public function showAction(Activity $activity)
     {
         $em = $this->getDoctrine()->getManager();
-
-        $regs = $em->getRepository(Registration::class)->findBy(['activity' => $activity, 'deletedate' => null, 'reserve_position' => null]);
+        $regs = $em->getRepository(Registration::class)->findBy(
+            ['activity' => $activity,
+            'deletedate' => null,
```

Comment:
We consider array instantiations as a function call, since the [..] operator is functionally equivalent to the array(..) function. Therefore, the first (key and) value of an array must be on a new intended line. In practice, this means the opening and closing square bracket may be placed on respectively the line before and after.